README.md

```diff
@@ -155,13 +168,54 @@ local_client = AsyncOpenAI(
 )
 
 docs = Docs(client=local_client,
-            embedding_model=LlamaEmbeddingModel(),
-            llm_model=OpenAILLMModel(config=dict(model="my-llm-model", temperature=0.1, frequency_penalty=1.5, max_tokens=512)))
+            docs_index=NumpyVectorStore(embedding_model=LlamaEmbeddingModel()),
+            texts_index=NumpyVectorStore(embedding_model=LlamaEmbeddingModel()),
+            llm_model=OpenAILLMModel(config=dict(model="my-llm-model", temperature=0.1, frequency_penalty=1.5, max_tokens=512)),
+        )
 ```
 
 ### Changing Embedding Model
 
-You can use langchain embedding models, or the [SentenceTransformer](https://www.sbert.net/) models. For example
+paper-qa defaults to using OpenAI (`text-embedding-3-small`) embeddings, but has flexible options for both vector stores and embedding choices. The simplest way to change an embedding is via the `embedding` argument to the `Docs` object constructor:
+
+```py
+from paperqa import Docs
+
+docs = Docs(embedding="text-embedding-3-large")
+```
+
+`embedding` accepts:
+
+- Any OpenAI embedding model name
+- [VoyageAI](https://docs.voyageai.com/docs/embeddings) model names (usable if `voyageai` is installed and `VOYAGE_API_KEY` is set)
+- `"sentence-transformers"` to use `multi-qa-MiniLM-L6-cos-v1` via [Sentence Transformers](https://huggingface.co/sentence-transformers)
+- `"hybrid-<model_name>"` i.e. `"hybrid-text-embedding-3-small"` to use a hybrid sparse keyword (based on a token modulo embedding) and dense vector embedding, any OpenAI or VoyageAI model can be used in the dense model name
+- `"sparse"` to use a sparse keyword embedding only
+
+For deeper embedding customization, embedding models and vector stores can be built separately and passed into the `Docs` object. Embedding models are used to create both paper-qa's index of document citation embedding vectors (`docs_index` argument) as well as the full-text embedding vectors (`texts_index` argument). They can both be specified as arguments when you create a new `Docs` object. You can use use any embedding model which implements paper-qa's `EmbeddingModel` class. For example, to use `text-embedding-3-large`:
+
+```py
+from paperqa import Docs, NumpyVectorStore, OpenAIEmbeddingModel
+
+docs = Docs(docs_index=NumpyVectorStore(embedding_model=OpenAIEmbeddingModel(name="text-embedding-3-large")),
```

Comment:
So long width README snippets are not easy to read because they require scrolling. Do you mind integrating https://github.com/adamchainz/blacken-docs into `pre-commit` for this?

And if you're tryna avoid doing too much here, feel free to ignore this comment

Reply:
this is a good call -- added